### PR TITLE
[DOCS][Layer Tests] Update layer test documentation

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Update layer test documentation. It will be used by external developers and mentioned in GFIs.

**Ticket:** TBD

### CODE REVIEW

File: tests/layer_tests/README.md
Comment (popovaan) [inline tests/layer_tests/README.md:5]:
```suggestion
1. Creation of a model with the tested operation using original framework API
```

**Review by popovaan — APPROVED** [review]

**Comment by popovaan** [comment]
Should we ask doc maintainers to review?

**Comment by rkazants** [comment]
> Should we ask doc maintainers to review?

I think we are doc maintainers for developers documentation:)
I wait for Maxim's response also.

File: tests/layer_tests/README.md
Comment (rkazants) [inline tests/layer_tests/README.md:32]:
```suggestion
Set environment variables `TEST_DEVICE` and `TEST_PRECISION` to select device and inference precision for OpenVINO inference.
Example to run the TensorFlow layer test for the `tf.raw_ops.Unique` operation on CPU with default inference precision:
```

**Review by mvafin — APPROVED** [review]

File: tests/layer_tests/README.md
Comment (rkazants) [inline tests/layer_tests/README.md:37]:
```suggestion
Example to run the TensorFlow layer test for the `tf.raw_ops.Unique` operation on CPU with default inference precision for device:
   ```sh
   export TEST_DEVICE="CPU"
   pytest tensorflow_tests/test_tf_Unique.py
   ```
Example to run the TensorFlow layer test for the `tf.raw_ops.Unique` operation on CPU and GPU with `FP16` and `FP32` inference precisions:
   ```sh
   export TEST_DEVICE="CPU;GPU"
   export TEST_PRECISION="FP32;FP16"
   pytest tensorflow_tests/test_tf_Unique.py
   ```

Comment (rkazants) [inline tests/layer_tests/README.md:37]:
```suggestion
Example to run the TensorFlow layer test for the `tf.raw_ops.Unique` operation on CPU with default inference precision:
   ```sh
   export TEST_DEVICE="CPU"
   pytest tensorflow_tests/test_tf_Unique.py
   ```
some
```

File: tests/layer_tests/README.md
Comment (mvafin) [inline tests/layer_tests/README.md:46]:
```suggestion
Example to run the PyTorch layer test for the `torch.linalg.cross` operation on GPU with `FP16` inference precisions:
   ```sh
   cd tests/layer_tests
   export TEST_DEVICE=GPU
   export TEST_PRECISION=FP16
   pytest pytorch_tests/test_cross.py
```
I wouldn't recommend running GPU with FP32, it won't work now, because it currently will only change thresholds, not actual precision.

**Pull request merged**
